app/models/concerns/scripts/import.rb

```diff
@@ -26,9 +39,9 @@ def import_zip zip_path
 
 
       def import_scripts scripts_data
-        scripts_data.each do |uuid, script_data|
-          import_script script_data, scripts_data
-        end
+        scripts_data.map do |uuid, script_data|
+          import_script(script_data, scripts_data)
```

Comment:
```suggestion
          import_script script_data, scripts_data
```
Cuando no es muy ambiguo lo dejamos sin los paréntesis. Quizás te retó rubocop por eso... jeje, pero si se puede y no queda extra raro vamos por no usarlos.

Reply:
Dale. Ahi los vuelo